### PR TITLE
OSDOCS 10276:Modified acme-cluster-issuer yaml file to add ingress class name

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.13+

Issue: [OSDOCS-10276](https://issues.redhat.com/browse/OSDOCS-10276)

Link to docs preview: [docs preview](https://75220--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/cert-manager-operator-issuer-acme.html#cert-manager-acme-http01_cert-manager-operator-issuer-acme)

SME review:
- [X] SME has approved this change.

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by anirudhAgniRedhat** [comment]
> /cc @anirudhAgniRedhat

looks good to me.

**Comment by mburke5678** [comment]
@subhtk A couple of comments. Otherwise LGTM

File: modules/cert-manager-acme-http01.adoc [hunk 1]
```diff
@@ -35,13 +35,23 @@ spec:
...
+          ingressClassName: openshift-default                                 <4>
 ----
 <1> Provide a name for the cluster issuer.
 <2> Replace `<secret_private_key>` with the name of secret to store the ACME account private key in.
 <3> Specify the URL to access the ACME server's `directory` endpoint. This example uses the _Let's Encrypt_ staging environment.
 <4> Specify the Ingress class.
 
+[NOTE]
+====
+If created without specifying `ingressClassName`, use the following command to patch the existing ingress:
```
[the remark this answers is not in the document]
Comment (subhtk) [reply modules/cert-manager-acme-http01.adoc:47]: ack.

File: modules/cert-manager-acme-http01.adoc [hunk 2]
```diff
@@ -35,13 +35,21 @@ spec:
...
 <3> Specify the URL to access the ACME server's `directory` endpoint. This example uses the _Let's Encrypt_ staging environment.
 <4> Specify the Ingress class.
 
+.. Optional: If you create the object without specifying `ingressClassName`, use the following command to patch the existing ingress:
+
+[source,terminal]
+----
+$ oc patch ingress/<ingress-name> --type=merge --patch '{"spec":{"ingressClassName":"openshift-default"}}' -n <namespace>
+----
+====
```
[the remark this answers is not in the document]
Comment (subhtk) [reply modules/cert-manager-acme-http01.adoc:51]: Ack.

File: modules/cert-manager-acme-http01.adoc [hunk 3]
```diff
@@ -35,13 +35,20 @@ spec:
...
-          class: openshift-default                                 <4>
+          ingressClassName: openshift-default                                 <4>
 ----
 <1> Provide a name for the cluster issuer.
 <2> Replace `<secret_private_key>` with the name of secret to store the ACME account private key in.
 <3> Specify the URL to access the ACME server's `directory` endpoint. This example uses the _Let's Encrypt_ staging environment.
 <4> Specify the Ingress class.
 
+.. Optional: If you create the object without specifying `ingressClassName`, use the following command to patch the existing ingress:
+
```
Comment (ShaunaDiaz) [inline modules/cert-manager-acme-http01.adoc:46]:
```suggestion
+
```
This is not aligned in the preview and needs an update before merge.

Comment (subhtk) [reply modules/cert-manager-acme-http01.adoc]: Ack.

**Pull request merged**